lib/dor/models/permissable.rb

```diff
@@ -0,0 +1,160 @@
+module Dor
+  module Permissable
+    extend ActiveSupport::Concern
```

Comment:
All new module that implement requirements from
https://consul.stanford.edu/display/chimera/Repository+Roles+and+Permissions
(Not entirely sure it needs to be a concern, just following the pattern of Governable, but this module has no database fields.  Comments or advice on this much appreciated.)